### ISSUE TITLE
Bundles not sent to the GW

### ISSUE THREAD

**Issue by demichele** [iot2tangle/Keepy#7]
Hey! 
I set buffer to TRUE and start sending datasets. After 10 datasets (the value by default) Keepy returns: 
All messages added with Channel ID: Malformed json - use iot2tangle json format

The database has the bundles but no channel id is associated. This is how one of the many row created look like: 
```
{"bundle":[{"iot2tangle":[{"sensor":"Environmental","data":[{"Pressure":"102033"},{"Temp":"26160"},{"Humidity":"33"}]},{"sensor":"Accel","data":[{"x":"9"},{"y":"12"},{"z":"1009"}]},{"sensor":"Gyroscope","data":[{"x":"122"},{"y":"1708"},{"z":"5246"}]},{"sensor":"Inertial","data":[{"x":"183"},{"y":"122"},{"z":"-915"}]},{"sensor":"Light","data":[{"milliLux":"192960"}]},{"sensor":"Magnetometer","data":[{"x":"-59"},{"y":"3"},{"z":"7"}]},{"sensor":"Acoustic","data":[{"mp":"0.004649"}]}],"device":"carpi","timestamp":"1601653408"},{"iot2tangle":[{"sensor":"Environmental","data":[{"Pressure":"102033"},{"Temp":"26160"},{"Humidity":"33"}]},{"sensor":"Accel","data":[{"x":"9"},{"y":"12"},{"z":"1009"}]},{"sensor":"Gyroscope","data":[{"x":"122"},{"y":"1708"},{"z":"5246"}]},{"sensor":"Inertial","data":[{"x":"183"},{"y":"122"},{"z":"-915"}]},{"sensor":"Light","data":[{"milliLux":"192960"}]},{"sensor":"Magnetometer","data":[{"x":"-59"},{"y":"3"},{"z":"7"}]},{"sensor":"Acoustic","data":[{"mp":"0.004649"}]}],"device":"carpi","timestamp":"1601653408"},{"iot2tangle":[{"sensor":"Environmental","data":[{"Pressure":"102033"},{"Temp":"26160"},{"Humidity":"33"}]},{"sensor":"Accel","data":[{"x":"9"},{"y":"12"},{"z":"1009"}]},{"sensor":"Gyroscope","data":[{"x":"122"},{"y":"1708"},{"z":"5246"}]},{"sensor":"Inertial","data":[{"x":"183"},{"y":"122"},{"z":"-915"}]},{"sensor":"Light","data":[{"milliLux":"192960"}]},{"sensor":"Magnetometer","data":[{"x":"-59"},{"y":"3"},{"z":"7"}]},{"sensor":"Acoustic","data":[{"mp":"0.004649"}]}],"device":"carpi","timestamp":"1601653408"},{"iot2tangle":[{"sensor":"Environmental","data":[{"Pressure":"102033"},{"Temp":"26160"},{"Humidity":"33"}]},{"sensor":"Accel","data":[{"x":"9"},{"y":"12"},{"z":"1009"}]},{"sensor":"Gyroscope","data":[{"x":"122"},{"y":"1708"},{"z":"5246"}]},{"sensor":"Inertial","data":[{"x":"183"},{"y":"122"},{"z":"-915"}]},{"sensor":"Light","data":[{"milliLux":"192960"}]},{"sensor":"Magnetometer","data":[{"x":"-59"},{"y":"3"},{"z":"7"}]},{"sensor":"Acoustic","data":[{"mp":"0.004649"}]}],"device":"carpi","timestamp":"1601653408"},{"iot2tangle":[{"sensor":"Environmental","data":[{"Pressure":"102033"},{"Temp":"26160"},{"Humidity":"33"}]},{"sensor":"Accel","data":[{"x":"9"},{"y":"12"},{"z":"1009"}]},{"sensor":"Gyroscope","data":[{"x":"122"},{"y":"1708"},{"z":"5246"}]},{"sensor":"Inertial","data":[{"x":"183"},{"y":"122"},{"z":"-915"}]},{"sensor":"Light","data":[{"milliLux":"192960"}]},{"sensor":"Magnetometer","data":[{"x":"-59"},{"y":"3"},{"z":"7"}]},{"sensor":"Acoustic","data":[{"mp":"0.004649"}]}],"device":"carpi","timestamp":"1601653408"}]} | Malformed json - use iot2tangle json format
```

If I get the complete bundle object from the database and send it to HTTP://GW_IP:8080/bundle_data it gets published to the Tangle.

**Comment by eduardopelitti** [comment]
Thanks for reporting! I will address the issue in the open PR #6

**Comment by eduardopelitti** [comment]
This should be fixed in [5d376c4](https://github.com/iot2tangle/Keepy/pull/6/commits/5d376c4fafb455e07848b541c8f6b884fb307921)
